integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -416,7 +416,6 @@ logserver:
   # dnsPolicy: ClusterFirst
   # JVM options specific to the logserver container
   jvmOptions:
-  nodeSelector: {}
```

Comment:
@jiacheliu3 I removed this because it was (and will still be) unused

Reply:
okay, let's deal with the `nodeSelector` in a separate PR after discussion

Reply:
Actually, let's keep it this way (don't remove this line).

One use case I can imagine is, there are nodes that have storage and nodes that do not. By defining a selector, the logserver is scheduled to a node that has storage, so that the logs are persisted to the storage.

Another argument is, let's do not take away what's there, unless the whole PR is intended for that. In this PR we add toleration, so let's not touch unrelated stuff. In the next PR just for nodeSelector, we can do the refactor as we redesign.

Reply:
@ZhuTopher I'll merge after this is reverted. Thanks!

Reply:
@ZhuTopher ooops sorry I seem to have clicked resolve unintendedly. Could you revert this line and then I'll merge? Thanks!

Reply:
Oh sorry, I didn't notice this comment previously my bad! The reason I removed this from the Helm chart `values.yaml` is because this field is unused in `templates/logserver/deployment.yaml`.

I tried to add the `nodeSelector` behaviour from our other templates to it but since we decided to leave it out of this PR I also removed the related chart value to avoid confusion (chart implies you _can_ specify a `nodeSelector` when you actually can't).

Anyway, again we can leave all this to a separate 'nodeSelector'-PR so I'll revert it for now.